### DISCUSSION TITLE
Running in production

### DISCUSSION THREAD

**Discussion by antisaa** [0xPolygon/polygon-edge#470] [Q&A] [answered]
I would like to use Polygon-edge framework to run a PoS chain in production. Which things should I look after? Is PoS even production ready?

Did you think about adding optionality to give validators in PoS addition rewards alongside of transaction fees?

**Comment by Kourin1996** [comment, marked as answer]
@antisaa Hello, Thank you for your interest and questions. Currently PoS is under development and there is a possibility of new features to be added. Please check future releases and Polygon Edge document. https://edge-docs.polygon.technology/docs/consensus/pos-concepts explains current PoS in Edge.

Now, rewards are paid by coin base only, regardless of the consensus you choose.
